File: Back/CedroRestaurante.Dominio/Entidades/Prato.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CedroRestaurante.Dominio.Entidades
{
    public class Prato : EntidadeBase
    {
        public string Nome { get; set; }
        public double Preco { get; set; }
        public int RestauranteId { get; set; }
        
        public Restaurante Restaurante { get; set; }
    }
}

File: Front/src/app/entities/prato.ts
export class Prato{
    id: number;
    nome: string;
    preco: number;
    restauranteId: number;

    constructor(init? : Partial<Prato>) {
        if(init != null) Object.assign(this, init);
        else{
            this.id = undefined;
            this.nome = undefined;
            this.preco = undefined;
            this.restauranteId = undefined;
        }
    }
}